src/sidebar/activity/util/activityUtil.ts

```diff
@@ -0,0 +1,69 @@
+import DateSlot from "../model/DateSlot";
+import localization from "../../../localization";
+
+const MS_IN_MINUTE = 1000 * 60; // eslint-disable-line @typescript-eslint/no-magic-numbers
+const MS_IN_HOUR = 60 * MS_IN_MINUTE; // eslint-disable-line @typescript-eslint/no-magic-numbers
+const MS_IN_DAY = 24 * MS_IN_HOUR; // eslint-disable-line @typescript-eslint/no-magic-numbers
+const MS_IN_WEEK = 7 * MS_IN_DAY; // eslint-disable-line @typescript-eslint/no-magic-numbers
+const MS_IN_MONTH = 30 * MS_IN_DAY; // eslint-disable-line @typescript-eslint/no-magic-numbers
+const MS_IN_YEAR = 365 * MS_IN_DAY; // eslint-disable-line @typescript-eslint/no-magic-numbers
+const AGO_YEAR_LIMIT = 2;
+
```

Comment:
Didn't test but this should work.
```suggestion
// eslint-disable @typescript-eslint/no-magic-numbers
const MS_IN_MINUTE = 1000 * 60;
const MS_IN_HOUR = 60 * MS_IN_MINUTE;
const MS_IN_DAY = 24 * MS_IN_HOUR;
const MS_IN_WEEK = 7 * MS_IN_DAY;
const MS_IN_MONTH = 30 * MS_IN_DAY;
const MS_IN_YEAR = 365 * MS_IN_DAY;
const AGO_YEAR_LIMIT = 2;
// eslint-enable @typescript-eslint/no-magic-numbers
```